test/mockupdb/operations.py

```diff
@@ -76,20 +76,6 @@
         reply={'cursor': {'id': 0, 'firstBatch': []}},
         op_type='may-use-secondary',
         not_master=not_master_reply_to_command),
```

Comment:
not_master_reply_to_command and not_master_reply_to_query need to be OpMsgReply replies.

Reply:
Done.